### PR TITLE
Update ucx_api.pyi stub

### PR DESCRIPTION
**Pull request by pentschev**
Update stub file `ucx_api.pyi` to reflect recent changes.

### CODE REVIEW

**Review by madsbk — APPROVED** [review]
LGTM, nice and clean :)

File: ucp/_libs/ucx_api.pyi [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import enum
-from typing import Callable, Iterable, Mapping, Optional
+from typing import Callable, Dict, Iterable, Mapping, Optional, Tuple
 
-def get_current_options() -> None: ...
+# typedefs.pyx
```
[the remark this answers is not in the document]
Comment (pentschev) [reply ucp/_libs/ucx_api.pyi:4]: I don't get it, why are you suggesting removing only this one?

[the remark this answers is not in the document]
Comment (madsbk) [reply ucp/_libs/ucx_api.pyi:4]: Ahh sorry my mistake, initially I though that it was some hanging debug code :)

[the remark this answers is not in the document]
Comment (madsbk) [reply ucp/_libs/ucx_api.pyi:4]: Please disregard

[the remark this answers is not in the document]
Comment (pentschev) [reply ucp/_libs/ucx_api.pyi:4]: No worries, thanks for clarifying!

**Comment by pentschev** [comment]
Thanks @madsbk for reviewing!

**Pull request merged**